vernac/g_vernac.ml4

```diff
@@ -78,9 +78,15 @@ GEXTEND Gram
     ]
   ;
   vernac:
+    [ [ IDENT "Unguarded";   (f, v) = vernac_loc -> (VernacGuarded false  :: f, v)
+      (* Disabled to not confuse the user: for the moment there is not way to disable guard check globally. *)
+      (* | IDENT "Guarded"; (f, v) = vernac_loc -> (VernacGuarded true :: f, v) *)
```

Comment:
Why don't you define this option? It should be really easy to add one. Indeed, this had me confused about the reasons for a three-valued flag.